prime-dotnet-webapi-tests/Unit Tests/SiteStatusStateEngine.Tests.cs

```diff
@@ -21,13 +21,31 @@ public static IEnumerable<object[]> SiteRegistrationActionData()
 
         [Theory]
         [MemberData(nameof(SiteRegistrationActionData))]
-        public void TestAllowableStatusChange_Active(SiteRegistrationAction action)
+        public void TestAllowableStatusChange_Actived(SiteRegistrationAction action)
```

Comment:
Should be `TestAllowableStatusChange_Archived`

Reply:
Correct. I just renamed the test methods to make more sense.